builder_data.yaml

```diff
@@ -58,7 +58,7 @@ yum_repo_command: |
     sudo chown -R root:release "${repodir}/repodata" ;
     sudo chmod -R g+w "${repodir}/repodata" ;
     createrepo --checksum=sha --database --update "${repodir}" ;
-    gpg --use-agent --armor --detach-sign -u __GPG_KEY__ "${repodir}/repodata/repomd.xml" ;
```

Comment:
How come specifying `--yes` and `--use-agent` broke? I don't see anything in the documentation that suggests they wouldn't work together.

Reply:
I honestly have no idea, but when I had them together, I got this error:
```
gpg: skipped "--yes": secret key not available
gpg: signing failed: secret key not available
```

Reply:
This led me to try removing `--use-agent`: https://github.com/IJHack/qtpass/issues/92#issuecomment-189268411